### DISCUSSION TITLE
Seat positioning

### DISCUSSION THREAD

**Discussion by YukoKyoden** [tsunamayo/Starship-EVO#4466] [Suggestions]
I've noticed in the past that there was a seat revision which was great.
The problem I'm seeing with the seats right now is we sit about 1 and a half (1/4th block's) in front of the back, as all seats are 3, deep and 3 tall. 

Would there be a possibility of reducing the seats to 2.5 deep by 3 tall to leave room for seat design but also so we can sit further up on the seats making it look and feel more natural?

**Comment by tsunamayo** [comment]
Hi, the problem is that it will break all existing blueprint and people would hate me for that... thx

**Reply by ZachZent** [reply]
Why not use keep the old seats as a hidden "legacy" item and just have the shorter seat going forward. That way it won't break anything since it isn't removed and the new chair can start replacing it.

**Reply by cloudrambler** [reply]
I have seen some great augmented seats in other peoples ship designs. Could we have a micro-seat block, so that we can make our own seats around it. Maybe a 0.25m cube, or even a slab. It could have an option for "sit on" or "it in front" to give choices for making the seat block visible for selection.
